commit 5cf0a610e8b54052d59fc848f36abcdcd1abe861
Author: Aric Cyr <aric.cyr@amd.com>
Date:   Mon Mar 8 13:10:50 2021 -0500

    drm/amd/display: 3.2.127
    
    This DC patchset brings improvements in multiple areas. In summary, we
    highlight:
    
    * Add debug out when viewport too small
    * use max lb for latency hiding
    * System black screen hangs on driver load
    * Fix UBSAN warning for not a valid value for type '_Bool'
    * Fix for outbox1 ring buffer typecasting issue
    * Bypass sink detect when there are no eDPs connected
    * Increase precision for bpp in DSC calculations
    * Add changes for dsc bpp in 16ths and unify bw calculations
    * Correct algorithm for reversed gamma
    * Remove MPC gamut remap logic for DCN30
    * Fix typo for helpers function name
    * Fix secure display lock problems
    * Fix no previous prototype warning
    * Separate caps for maximum RGB and YUV plane counts
    * Add debugfs to control DMUB trace buffer events
    * [FW Promotion] Release 0.0.56
    * DCHUB underflow counter increasing in some scenarios
    * fix dml prefetch validation
    * fix dcn3+ bw validation soc param update sequence
    * add a func to disable accelerated mode
    * Fix potential memory leak
    
    Tested-by: Daniel Wheeler <daniel.wheeler@amd.com>
    Signed-off-by: Aric Cyr <aric.cyr@amd.com>
    Acked-by: Solomon Chiu <solomon.chiu@amd.com>
    Signed-off-by: Alex Deucher <alexander.deucher@amd.com>

diff --git a/drivers/gpu/drm/amd/display/dc/dc.h b/drivers/gpu/drm/amd/display/dc/dc.h
index d26472ef1572..352651c805ff 100644
--- a/drivers/gpu/drm/amd/display/dc/dc.h
+++ b/drivers/gpu/drm/amd/display/dc/dc.h
@@ -45,7 +45,7 @@
 /* forward declaration */
 struct aux_payload;
 
-#define DC_VER "3.2.126.1"
+#define DC_VER "3.2.127"
 
 #define MAX_SURFACES 3
 #define MAX_PLANES 6